src/utilities.js

```diff
@@ -107,7 +107,10 @@ export function getRequiredCombatSkill(skillConstant) {
             skill = "magic";
```

Comment:
These can all be returned instead of setting "skill" then returning it since we're using a default now

Reply:
[Updated](https://github.com/EthanVieira/rs-clicker/pull/51/commits/1e14e04e305e70c3386791a2c9308c511e85bc7a).